pallets/msa/src/lib.rs

```diff
@@ -297,6 +321,31 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Adds an association between MSA id and ProviderMetadata. As of now, the
+		/// only piece of metadata we are recording is provider name.
+		///
+		/// ## Errors
+		/// - Returns
+		///   [`DuplicateProviderMetadata`](Error::DuplicateProviderMetadata) if there is already a ProviderMetadata associated with the given MSA id.
+		#[pallet::weight(T::WeightInfo::register_provider())]
+		pub fn register_provider(origin: OriginFor<T>, provider_name: Vec<u8>) -> DispatchResult {
```

Comment:
Don't forget to add this to the api-augment code

Reply:
I updated `lookup.ts`. Are there any other files I should be concerned about updating?